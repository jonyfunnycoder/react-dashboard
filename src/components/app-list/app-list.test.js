import React from 'react';
import renderer from "react-test-renderer";
import AppList from "./app-list";
import AppListImage from "./app-list-image";
import AppListBage from "./app-list-badge";
import Avtar from "../../images/avtar.png";

test('renders app list with icon and badge correctly ', () => {
    const tree = renderer
        .create(
            <AppList
                ListIcon={<AppListImage url={Avtar} />}
                Badge={<AppListBage value={6788} />}
            />
        )
        .toJSON();

    expect(tree).toMatchSnapshot();
});
