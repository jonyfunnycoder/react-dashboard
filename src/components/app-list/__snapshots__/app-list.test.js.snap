// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app list with icon and badge correctly  1`] = `
<div
  className="app-list-item"
  onClick={[Function]}
>
  <div
    className="list-item-left"
  >
    <img
      alt="test"
      className="avtar"
      src="avtar.png"
    />
    <label />
  </div>
  <div
    className="list-badge"
  >
    6788
  </div>
</div>
`;
